server.js

```diff
@@ -37,6 +38,9 @@ const queue = new AwaitQueue()
 // @type {Map<Number, Room>}
 const rooms = new Map()
 
+// Maps of eventNotificationUri
+const notificationsUri = new Map()
```

Comment:
```suggestion
const notificationUris = new Map()
```

The plural should be in "URIs"

Reply:
Resolved